plugins/modules/imc_rest.py

```diff
@@ -258,12 +258,13 @@
   description: RAW XML output received from the Cisco IMC, with error details.
   returned: failed
   type: str
-  sample: >
+  sample: |
```

Comment:
I think this was intentionally `>` and not `|`.
```suggestion
  sample: >
```